homeassistant/components/roomba/translations/en.json

```diff
@@ -39,14 +32,10 @@
             },
             "user": {
                 "data": {
-                    "blid": "BLID",
-                    "continuous": "Continuous",
-                    "delay": "Delay",
-                    "host": "Host",
-                    "password": "Password"
+                    "host": "Host"
                 },
-                "description": "Currently retrieving the BLID and password is a manual process. Please follow the steps outlined in the documentation at: https://www.home-assistant.io/integrations/roomba/#retrieving-your-credentials",
-                "title": "Connect to the device"
```

Comment:
This is changing translations, that doesn't work from PRs and can only be done from Lokalise.